Makefile

```diff
@@ -1,41 +1,89 @@
-# Build the docker images contained in this repo
+# Makefile
+# Build and run local stack for Pyronear
+
+SHELL := /bin/sh
+COMPOSE := docker compose
```

Comment:
I find it a bit overkill to have this alias as is does not help with the action behind each make command and the benefit is minimal. Is this in case the docker compose command changes one day?